src/scss/components/_main.scss

```diff
@@ -0,0 +1,30 @@
+@import '../variables';
+
+.main {
+  height: 75vh;
+  width: 70vw;
+  margin:50px auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  color: $info;
+  background-color: $middle;
+  border-radius: 100px;
+
+span{
+  opacity: 0.9;
+  font-size: 35px;
+  font-weight: 700;
+  color: $info;
+  letter-spacing: 0.3em;
```

Comment:
why

Reply:
the answer is easy to see, because I chose to span a special look on the main page